ci/jenkins/jobs.yaml

```diff
@@ -81,6 +82,8 @@
     jobs:
         - '{name}/{platform}/{test}-daily'
         - '{name}/{platform}/update-daily'
+        - '{name}/{platform}/trigger-upgrade-from-4-2-daily'
+        - '{name}/{platform}/upgrade-from-4-2-daily'
```

Comment:
I am not sure I understand this differentiation between `trigger-upgrade` and `upgrade`. One is creating the worker and the other one using the created worker to actually run the upgrade test? I find it a bit confusing. Is this done to avoid having a very complex node setup (a setup that includes creating the node)?

Reply:
Yes,it is to avoid a massive complex job that takes care of everything. I think it is better to split the process int two. Each job has a well-defined purpose. Besides, the upgrade can be tested in isolation.